addons/binding/org.openhab.binding.sensebox/src/main/java/org/openhab/binding/sensebox/config/SenseBoxConfiguration.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.sensebox.config;
+
+/**
+ * The {@link SenseBoxConfiguration} is the base class for configuration
+ * information held by devices and modules
+ *
+ * @author Hakan Tandogan - Initial contribution
+ */
+public class SenseBoxConfiguration {
+
+    private String thingUid;
```

Comment:
The thingUID is not a config parameter of the Thing, so it is rather strange to have that added here - it really doesn't belong in here.